tests/ui_tests/app/page/admin/page.py

```diff
@@ -524,47 +524,50 @@ class AdminPoliciesPage(GeneralAdminPage):
         CommonTable.header,
     ]
 
-    def click_create_policy_btn(self):
+    def open_create_policy_popup(self):
         self.find_and_click(AdminPoliciesLocators.create_policy_btn)
         self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.block)
 
     @allure.step('Create new policy')
     def create_policy(
         self, policy_name: str, description: Optional[str], role: str, users: Optional[str], groups: Optional[str]
     ):
-        self.click_create_policy_btn()
+        assert users or groups, "There are should be users or groups in the policy"
+        self.open_create_policy_popup()
         self.send_text_to_element(AdminPoliciesLocators.AddPolicyPopup.FirstStep.name_input, policy_name)
         if description:
             self.send_text_to_element(AdminPoliciesLocators.AddPolicyPopup.FirstStep.description_input, description)
         with allure.step(f"Select role {role} in popup"):
             self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_select)
             self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item)
-            for available_role in self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item):
+            available_roles = self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item)
+            assert role in [a.text for a in available_roles], f"There are no role {role} in select role popup"
+            for available_role in available_roles:
                 if available_role.text == role:
                     available_role.click()
                     break
-        assert users or groups, "There are should be users or groups in the policy"
+
+        def fill_users_or_group_select(items, available_items):
+            for item in items.split(","):
+                assert item in [u.text for u in available_items], f"There are no {item} in select role popup"
+                for available_item in available_items:
+                    if available_item.text == item:
+                        self.scroll_to(available_item)
+                        available_item.click()
```

Comment:
I think `for ... else` will fit nice here

Reply:
changed